src/ui/screens/RegistrationForm.tsx

```diff
@@ -82,7 +82,7 @@ export default class RegistrationForm extends React.Component<Props, State> {
         return key.dataType === "date";
     }));
     _.each(dateFields, (field) => {
-        dates[field] = String(Date.parse(data[field]));
+        dates[field] = String(Math.round(Date.parse(data[field])/1000));
```

Comment:
`Date.parse` can throw an exception if the format can't be formatted to anything meaningful. You might want to use `TryParse` to prevent from that occurrence. See this doc link for more info: https://docs.microsoft.com/en-us/dotnet/api/system.datetime.tryparse?view=net-5.0#System_DateTime_TryParse_System_String_System_DateTime__

Reply:
That's .NET, though - is there a similar method in JS?

Reply:
Blimey! Yes. Yes, parseInt will be the way to go on JS. Is important to know wether the conversation fail. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
parseInt parses strings into integers.  What this is doing is parsing a Number into a Date type, so parseInt probably wouldn't be the right function.  I edited this and tested it with the string "banana" to see the failure behavior.  It works to save the credential with the error code, if that's what we want the fail case to be.  Essentially though the Date.parse function will return NaN if the expected Number param is not a Number.

Reply:
I think this implementation works - didn't realize `Date.parse` doesn't throw an error for invalid inputs, but I think the error case is handled here